packages/web3-utils/src/formatter.ts

```diff
@@ -74,11 +72,7 @@ const findSchemaByDataPath = (
 			result = (result.properties as Record<string, JsonSchema>)[dataPart];
 		} else if (result.items && (result.items as JsonSchema).properties) {
 			const node = (result.items as JsonSchema).properties as Record<string, JsonSchema>;
-
-			if (!node) {
```

Comment:
we remove this
this conditional can never be met, 
`} else if (result.items && (result.items as JsonSchema).properties) {`
we make sure `result.properties` always exists